Userland/Libraries/LibMarkdown/Text.cpp

```diff
@@ -663,14 +663,26 @@ NonnullOwnPtr<Text::Node> Text::parse_link(Vector<Token>::ConstIterator& tokens)
     };
 
     StringBuilder address;
+    DeprecatedString href;
     for (auto iterator = tokens + 1; !iterator.is_end(); ++iterator) {
         // FIXME: What to do if there's multiple dimension tokens?
         if (is_image && !address.is_empty() && parse_image_dimensions(iterator->data))
             continue;
 
         if (*iterator == ")"sv) {
             tokens = iterator;
-            return make<LinkNode>(is_image, move(link_text), address.to_deprecated_string().trim_whitespace(), image_width, image_height);
+
+            href = address.to_deprecated_string().trim_whitespace();
+
+            // Add file:// if the link is an absolute path otherwise it will be assumed relative.
+            if (AK::StringUtils::starts_with(href, "/"sv, CaseSensitivity::CaseSensitive)) {
+                StringBuilder href_builder;
+                href_builder.append("file://"sv);
+                href_builder.append(href);
+                href = href_builder.to_deprecated_string();
```

Comment:
You can simplify this to:
```suggestion
                href = DeprecatedString::format("file://{}", href);
```
Then you wouldn't need the `{}` for the `if` either.

Reply:
@AtkinsSJ It appears that format is not a member of AK::DeprecatedString. Did you mean something else?

Reply:
Maybe `formatted`?

Reply:
Ah yeah, that's the one sorry!